cvat/apps/engine/static/engine/js/shapeCreator.js

```diff
@@ -425,7 +441,7 @@ class ShapeCreatorView {
                     this._controller.finish(box, this._type);
                 }
 
-                this._controller.switchCreateMode(true);
+                this._controller.switchCreateMode(true, usingShortkey);
```

Comment:
Always false is suitable here, I assume. 
And you do not need an additional argument :)